archiver.go

```diff
@@ -107,6 +121,7 @@ func (a *Archiver) do(log log15.Logger, j *Job) (err error) {
 			finalErr = ErrClone.Wrap(err, endpoint)
 		}
 
+		r.Status = model.Pending
```

Comment:
If we do, we'd need another method to update when is not found. Right now, `dbUpdateFailedRepository` also saves the repository when it's status is `NotFound`